### PR TITLE
Document new associate methods

### PR DESCRIPTION
**Pull request by mj850**
## What is the purpose of the change?

Add documentation about the new associate methods in the addr precompile

## Describe the changes to the documentation
Documentats the new associate methods in the addr precompile, including examples for how to use them.

## Notes
-

### CODE REVIEW

**Review by besated — APPROVED** [review]
i might've missed it, but does this include the method to associate via pubkey?

File: pages/dev-interoperability/precompiles/addr.mdx [hunk 1]
```diff
@@ -11,6 +11,64 @@ Association takes place when the wallet first signs and broadcasts any transacti
...
+  const request = {
+      r: numberToHex(r),
+      s: numberToHex(s),
+      v: numberToHex(v - 27),
+      custom_message: customMessage
+  };
+  await contract.associate(request.v, request.r, request.s, request.custom_message)
+  ``` 
+
+- `sendNative`: Enables a user to send native Sei to another user.
```
Comment (dssei) [inline pages/dev-interoperability/precompiles/addr.mdx]: typo?

File: pages/dev-interoperability/precompiles/addr.mdx [hunk 2]
```diff
@@ -11,6 +11,64 @@ Association takes place when the wallet first signs and broadcasts any transacti
 
 ## Functions
 
+### Transactions
+- `associate`: Associates an EVM address with it's corresponding Cosmos address on chain using a signature. For association via a pubkey, use `associatePubKey` instead.
```
Comment (dssei) [inline pages/dev-interoperability/precompiles/addr.mdx]: nit: Cosmos -> Sei native

**Comment by cordt-sei** [comment]
> i might've missed it, but does this include the method to associate via pubkey?

@besated yes I asked the same thing because it looks so familiar. It is basically the same as before, except you can do it for the other wallet without that wallet having 1 wei now.

**Review by cordt-sei — APPROVED** [review]

**Pull request merged**